### DISCUSSION TITLE
error executing dbeaver zipped version in windows 10

### DISCUSSION THREAD

**Discussion by verajosemanuel** [dbeaver/dbeaver#10741] [Q&A] [answered]
#### System information: 
- windows 10
- DBeaver version 7.3 (zip)

#### Describe the problem you're observing:
Dialog error: 
![dbeavererror](https://user-images.githubusercontent.com/15836677/102712883-8528d100-42c4-11eb-9fb9-913c6c4e5d21.png)

#### Steps to reproduce, if exist:

1. Uncompressed dbeaver zip version
2. double click dbeaver.exe
3.  error dialog

Tried to locate 32 bits version just in case....to no avail

**Comment by SubOptimal** [comment]
Based on the path `C:\Progam Files (x86)\` I assume you have a 32 bit Java installed.

- Could you install a 64 bit Java version to see if this would solve your issue.

- Which Java version do you have installed?
- Based on https://www.oracle.com/java/technologies/javase/javase9-archive-downloads.html the last 32 bit Java version from Oracle seems to be < Java 9. To run dbeaver you would need Java >= 11.

**Comment by verajosemanuel** [comment]
I have no privileges to install any software on this machine. I hoped I could use portable dbeaver. Java Version is 12

**Reply by SubOptimal** [reply]
If you can't install software, but are able to use a portable version. You might have a look at the portable (ZIP)  version of AdoptOpenJDK
https://adoptopenjdk.net/releases.html?variant=openjdk11&jvmVariant=hotspot

- select
-- Operating System: Windows
-- Architecture: x64
- chose JRE (runtime) or JDK (for development) based on your needs
- download the ZIP archive and unzip it
- dbeaver needs to know about the location of this Java installation

(following steps are currently not tested)
- assuming that you downloaded `OpenJDK11U-jre_x64_windows_hotspot_11.0.9.1_1.zip` and it was extracted to `c:\progs\jdk-11.0.9.1+1-jre` 
- create a small batch file (e.g. `c:\progs\dbeaver.cmd`) with following content

```
REM set the environment variables
set JAVA_HOME=c:\progs\jdk-11.0.9.1+1-jre
set PATH=%JAVA_HOME%\bin;%PATH%

REM change to the dbeaver location
cd c:\Users\e106483\Downloads\dbeaver-ce-6.1.4-win32.win32.x86_64\dbeaver\

REM run dbeaver
.\dbeaver.exe
```

**Comment by kseniiaguzeeva** [comment]
@SubOptimal was right. DBeaver doesn't support 32-bit Java

**Reply by verajosemanuel** [reply]
Thanks @SubOptimal , I will try

**Comment by serge-rider** [comment, marked as answer]
Since 7.3.1 DBeaver is always packaged with Java 11. No need to install it separately.
